test/integration/admin_can_create_store_items_test.rb

```diff
@@ -16,7 +16,7 @@ class AdminCanCreateStoreItemsTest < ActionDispatch::IntegrationTest
     fill_in 'Category', with: @category.id
     fill_in 'Price', with: '777'
     click_button 'Create Item'
-    assert page.has_content?('Item has been added to store')
+    assert page.has_content?('Item NewGear has been added to store')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.